resources/domain/account/index.ts

```diff
@@ -1,3 +1,11 @@
+import { getSignerDisplayType } from "../signer"
+
+export const accountNS = '114c39e5-cd7d-416f-ab9e-5ab6ab0218ce'
+
+export const isDefaultAccountName = ({ name, lastSignerType }: Account) => name.toLowerCase() === getDefaultAccountName(lastSignerType)
```

Comment:
we should probably discuss this approach in general since last signer type can diverge from the account name. if you have two signers that can sign for the same account this will break if you've most recently used the one that wasn't used to initially create the account

Reply:
Discussed and sticking with the original implementation

Reply:
I think what we discussed was not saving the account name if it's _any_ default name, not just the one for the most recent signer. this accounts for the scenario where you create an account using, say, a Ledger, then switch to using a Trezor or hot signer for the same account. in that case we would save the account name as "Trezor account" even though I don't think that's the intention.